src/Resources/contao/library/Contao/Combiner.php

```diff
@@ -73,12 +73,29 @@ class Combiner extends \System
 	 */
 	protected $arrFiles = array();
 
+	/**
+	 * Webdir relative to TL_ROOT
+	 * @var string
+	 */
+	protected $strWebDir;
+
 
 	/**
 	 * Public constructor required
 	 */
 	public function __construct()
 	{
+		$strWebDir = \System::getContainer()->getParameter('contao.web_dir');
+
+		if (strncmp($strWebDir, TL_ROOT . '/', strlen(TL_ROOT . '/')) === 0)
```

Comment:
I think this should be `DIRECTORY_SEPARATOR`, because `kernel.root_dir` will be `C:\…` on Windows.